tests/run/binop_reverse_methods_GH2056.pyx

```diff
@@ -166,6 +172,7 @@ class OverloadCApi(Base):
         self.derived_implemented = derived_implemented
 
     def __add__(self, other):
+        assert(cython.typeof(self) != "OverloadCApi")  # should be untyped
```

Comment:
:) You probably didn't notice, but you're (nicely) using spaces around the comparison operator here but not in all other places.

Also note that `assert` is not a function in Python but a statement. It doesn't need parentheses.